### ISSUE TITLE
Placeable trigger_catapult for fling correction

### ISSUE THREAD

**Issue by The-Queen55142** [BEEmod/BEE2-items#2795]
So this is an Invisible item that uses faithplate trajectory. And when used. It acts like a magnet for flinging/midair Players. That way if we were to have a really difficult Fling in a puzzle. It's not impossible because the magnet will help. This also would help with Faith Plate maps like Portal 2's Test Chamber 6 (after GLaDOS awakens.) Where you go through Flip panels. so you can hit the next plate. orr… next Gel bounce.

---

*Additional notes from duplicate issue:*

In the editor, it would function like the faith plate item, except it is invisible, represented by a trigger block in the editor. They do not produce a bullseye on targeted surfaces.

The trigger would be mounted on a surface(preferably on a portalable surface) and would only enable if a portal is placed on the same tile as the trigger_catapult item.

It could feature a "Button Type" dropdown menu, with each option allowing the user to configure what the trigger_catapult affects.

* Weighted: Players and objects(cubes, turrets, gel bombs) are both affected.
* Cube: Only players are affected, cubes/turrets/gel bombs are unaffected by the catapult.
* Sphere: Only objects are affected, players are not affected by the catapult.

**Comment by ZipZoptheCat** [comment]
It's true that it often takes a lot of trial and error to get the map the right shape for a fling. Sometimes dozens of compiles with minor changes.

**Comment by Nick2007Portal** [comment]
Not really possible, since did not gaven more explanations.

**Comment by CTB33391** [comment]
So, an invisible item that uses trajectory to help make flings easier by adjusting velocity.
For example, if there was a propulsion-repulsion leap into a portal, the magnet would correct you towards where the portal is. Don't see much difficulty adding this: just a trigger_push that is adjusted with an (invisible in-game) Faith Plate bulls-eye.

**Comment by King0191** [comment]
Yes, exactly, CTB. Plus its pretty much literally what I said. I don't understand why Nick thought its not possible but eh

**Comment by Endermage77** [comment]
It's basically what the game already does to you when you drop into floor portals: It centers you into the target if you're slightly off course so you don't mess up by tiny amounts.

**Comment by King0191** [comment]
Pretty much. But with this it happens in different circumstances.

**Comment by Endermage77** [comment]
Ja.

**Comment by TeamSpen210** [comment]
Why? It's obvious how it works.

**Comment by SCIENTIST2004** [comment]
![player magnet](https://user-images.githubusercontent.com/35931028/44076576-59eaede0-9fa9-11e8-80eb-0349eb995082.png)

**Comment by Endermage77** [comment]
In layman terms

If player is going to miss target slightly then
Drag the player to the middle of the target mid-flight.

**Comment by HugoBDesigner** [comment]
Maybe this could replace #2722 ?

**Comment by HugoBDesigner** [comment]
Actually, it'd only replace #2722 if it was possible to make this work for items too, and not just players. But then it'd change the core purpose of this item.

**Comment by King0191** [comment]
Updated the post with new information and updates after the conversations from earlier

**Comment by Luke18033** [comment]
This should probably only be the size of a 1/4 block (or even 1/8), and would have velocity check enabled so just walking into it wouldn't trigger the fling.

**Comment by Luke18033** [comment]
And it should be able to attach to panel items the way placement helpers do.

**Comment by Konclan** [comment]
Moving to #3886
